src/utils/sort-guide-list.js

```diff
@@ -0,0 +1,19 @@
+const list = [
+  'Citra and 3DS',
+  'Dolphin',
+  'Ruby, Sapphire, and Emerald',
+  'FireRed and LeafGreen',
+  'Diamond, Pearl, and Platinum',
+  'HeartGold and SoulSilver',
+  'Black and White',
+  'Black 2 and White 2',
+  'X and Y',
+  'Omega Ruby and Alpha Sapphire',
+  'Sun and Moon',
+  'Ultra Sun and Ultra Moon',
+  'Sword and Shield',
+];
+
+export const sortList = (first, second) => {
+  return list.indexOf(first) - list.indexOf(second);
+};
```

Comment:
This error will prevent building the site if a category is added or renamed without being added to the sorting list.  During development, the error will show up in place of the site to let the developer or guide editor know the list should be updated.

```suggestion
export const sortList = (first, second) => {
  const firstIndex = list.indexOf(first);
  const secondIndex = list.indexOf(second);

  if (firstIndex === -1 || secondIndex === -1) {
    // This will stop the Gatsby release build or show an error during development
    // to make sure unordered categories won't deploy.
    throw new Error(
      'Unexpected category!  Please ensure all guide categories are in the sorted category list.',
    );
  }

  return firstIndex - secondIndex;
};
```